.github/workflows/build.yml

```diff
@@ -298,6 +298,39 @@ jobs:
       - name: Fail if any tests failed
```

Comment:
Ignore the spot where the comment is, but we need to add `chrome-optimism-e2e-send` to `cleanup > needs` array